BurnOutSharp/ProtectionType/BDPlus.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace BurnOutSharp.ProtectionType
+{
+    public class BDPlus
+    {
+		// TODO: Detect version
+        public static string CheckPath(string path, IEnumerable<string> files, bool isDirectory)
+        {
+            if (!isDirectory)
+                return null;
+
+            if (Directory.Exists(Path.Combine(path, "BDSVM")))
```

Comment:
Does this folders existence determine that the protection exists? Or does it require having something in it? For example, this would be the case for seeing if a disc is DVD-Audio or DVD-Video.